ansible/group_vars/all/h5p-statics.yml

```diff
@@ -0,0 +1,3 @@
+H5P_STATICFILES_SERVER_IMAGE: quay.io/schulcloudverbund/h5p-staticfiles-server
+H5P_STATICFILES_SERVER_IMAGE_TAG: 0.1.0
```

Comment:
this must be to dof_app_deploy and then to the ansible host_vars for each instance at the version.yml

Reply:
The `H5P_STATICFILES_SERVER_IMAGE_TAG` is the topic from my comment